core/src/com/unciv/logic/civilization/CivilizationInfo.kt

```diff
@@ -622,6 +632,11 @@ class CivilizationInfo {
             cityInfo.civInfo = this // must be before the city's setTransients because it depends on the tilemap, that comes from the currentPlayerCivInfo
             cityInfo.setTransients()
         }
+
+        passThroughImpassableUnlocked = passableImpassables.isNotEmpty()
+        // All civs normally take 50 damage on a mountain. Cache whether this civ is an exception for performance reasons.
+        nonStandardTerrainDamage = getMatchingUniques("Units ending their turn on a [] take [] damage")
+            .any { it.params[0] != Constants.mountain || it.params[1].toInt() != 50 }
```

Comment:
Not sure how relevant this really is...

Reply:
It's to avoid doing string comparisons during pathfinding.